spring-boot-project/spring-boot/src/test/java/org/springframework/boot/SpringApplicationTests.java

```diff
@@ -36,6 +36,7 @@
 import org.mockito.ArgumentMatchers;
 import org.mockito.InOrder;
 import org.mockito.Mockito;
+import org.springframework.core.env.*;
```

Comment:
We prefer not to use wildcard imports. Could you please remove this and reinstate the individual imports below?

Reply:
Oh -- sorry. IntelliJ just did that automatically. I didn't notice. Stupid IDEs. Fixed now.

Reply:
@AaronTraas you can configure IJ not to do that in Preferences (Editor > Code Style > Java > Imports and increase the class/name count to something ridiculously high :) )

Reply:
@snicoll Yeah, I discovered that before committing. Set it to 9999. I think that should have me covered :)